docs/tutorials/api/datasets.ipynb

```diff
@@ -0,0 +1,877 @@
+{
```

Comment:
<p>I think you could discuss joint vs stacked analysis and present <code>Datasets</code>  as the tool to handle the former.</p><p> </p><p>I would present some of the functionalities here: I/O of course but also stack_reduce and maybe some others.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy/pull/3488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>